encrypted-media/content/content-metadata.js

```diff
@@ -11,6 +12,7 @@ content = addMemberListToObject( {
                     },
 
     'mp4-av-multikey' : {
+                        assetId:        'mp4-basic',
```

Comment:
No, at present we are using the same asset for this test as for the basic playback test (one asset with 2 keys). So this is the correct that the assetId in DRMToday for this test case is 'mp4-basic'